openedx/adg/common/mailchimp_pipeline/client.py

```diff
@@ -0,0 +1,124 @@
+"""
+Mailchimp Connection Client
+This code is used to sync users with mailchimp subscriber list
+"""
+import hashlib
+import json
+
+from django.conf import settings
+from requests import request
+from requests.exceptions import HTTPError
+
+
+class MailChimpException(Exception):
+    pass
```

Comment:
I know we benchmarked the code in PhilU. But I guess we should not override the base `Exception` if are not actually overriding or extending anything in it.